SETUP.md

```diff
@@ -0,0 +1,39 @@
+# Nylas Use Case Samples
+
+**⚠️ Warning: This repo is not meant for use in production and stability is not fully guaranteed.**
+
+Explore working front-end and back-end sample code using an integration with Nylas.
+
+The sample application redirects to a prebuilt authentication page hosted on Nylas, then allows that authenticated user to interact with Nylas API from a web client.
+
+## ⚙️ Environment Setup
+
+Let’s check that our environment is set up to use the [Nylas Node SDK](https://github.com/nylas/nylas-nodejs). Check the Node version in your terminal:
+
+```bash
+$ node -v
+v18.0.0
+```
```

Comment:
If the user is truly a beginner, then they might find the bash script confusing (they may not understand what `$` is), for example.

```suggestion
The [Nylas Node SDK](https://github.com/nylas/nylas-nodejs) requires Node version 18.0.0. Check your version in your terminal:

```bash
node -v
```
```